server/server.go

```diff
@@ -0,0 +1,168 @@
+package server
+
+import (
+	"fmt"
+	"net"
+	"context"
+	"strings"
+	"romano.com/proto"
+	"romano.com/mongo"
+	"romano.com/model/fileModel"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type Server struct {
+	port string
+}
+
+const (
+	mongoID string = "_id";
+)
+
+var mongoManager mongo.Controller;
+
+func New(port string) Server {
+	var s Server;
+	s.port = port;
+	mongoManager = mongo.NewController("mongodb://mongo:27017");
+  	return s;
+}
+
+func (s *Server) CreateListener() {
+
+	listener, err := net.Listen("tcp", s.port);
+	if err != nil {
+		panic(err)
+	}
+
+	srv := grpc.NewServer();
```

Comment:
The gRPC server and its services should be defined and registered before you start listening.